Bottom: 32d82a8bec31f1a184da6158fb384e7b1efa152c
Top:    2f56e0464b591a6fd34365960ff90c801d283d20
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 01:56:06 +0000

crosshair.c: When drawing via DRC clearence, pass correct angle.

360 * 64 was a hold-over from before the HID split. The passed
angle should be 360.




---

diff --git a/src/crosshair.c b/src/crosshair.c
index 2996516..8321929 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -593,7 +593,7 @@ DrawAttached (Boolean BlockToo)
 	  s = Settings.ViaThickness / 2 + PCB->Bloat + 1;
 	  gui->set_color (Crosshair.GC, Settings.CrossColor);
 	  gui->draw_arc (Crosshair.GC,
-			 Crosshair.X, Crosshair.Y, s, s, 0, 360 * 64);
+			 Crosshair.X, Crosshair.Y, s, s, 0, 360);
 	  gui->set_color (Crosshair.GC, Settings.CrosshairColor);
 	}
       break;
